test/LifToken.js

```diff
@@ -79,9 +79,10 @@ contract('LifToken', function(accounts) {
 
     var dataEncoded = help.hexEncode(JSON.stringify({awesomeField:"AwesomeString"}));
     let transaction = await token.transferData(accounts[2], help.formatBalance(1000), dataEncoded, false, {from: accounts[1]});
+    let decodedEvents = help.abiDecoder.decodeLogs(transaction.receipt.logs);
```

Comment:
I'll try to learn more on my own, but could you explain briefly what's the difference between `transaction.logs` and `transaction.receipt.logs`?